taxonomy-editor-frontend/src/pages/project/search/ThreeOptionsSwitch.tsx

```diff
@@ -0,0 +1,76 @@
+import { ToggleButton, ToggleButtonGroup, Typography } from "@mui/material";
+import React, { Dispatch, SetStateAction } from "react";
+
+const toggleTheme = {
+  "& .MuiToggleButton-root": {
+    color: "#341100",
+    "&.Mui-selected": {
+      color: "#341100",
+    },
+    borderColor: "rgba(0, 0, 0, 0.23)",
+  },
+};
```

Comment:
nit: If you must...
I would prefer if possible to add a global colour theme using MUI's theming options for example (https://mui.com/material-ui/customization/theming/)
NB: the fact that you need to do that to customise MUI's colours is why I push for the use of `shadcn/ui` (https://ui.shadcn.com) for example - with MUI you do not own your component's library